src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -1134,9 +1137,10 @@ def create() -> QueryInfo:
         table = make_table()
         query_name = f"ucx_query_Q{make_random(4)}"
         query = ws.queries.create(
-            name=f"{query_name}",
+            name=query_name,
             description="TEST QUERY FOR UCX",
             query=f"SELECT * FROM {table.schema_name}.{table.name}",
+            tags=[query_name],
```

Comment:
Why tag the query with the query name? Looks redundant

Reply:
this is so wethis gives us a query with a tag, to verify our tag changes work correctly - could just use any random string so this is cleanest

Reply:
I would prefer that yes, maybe make it more clear with a "original tag" or "tag before migrating" or something similar (+ random string)

Reply:
done